extensions/trello/package.json

```
@@ -0,0 +1,68 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "trello",
+  "title": "Trello",
+  "description": "Raycast extension for showing, searching, and interacting with cards from Trello",
+  "icon": "command-icon.png",
+  "author": "ChrisChinchilla",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "Fetch Todos",
+      "subtitle": "Trello",
+      "description": "Fetch all assigned todos from Trello",
+      "mode": "view"
+    }
+  ],
+  "preferences": [
+    {
+      "name": "username",
+      "type": "textfield",
+      "required": true,
+      "title": "Trello username",
+      "description": "Trello username",
```

```suggestion
      "title": "Trello Username",
      "description": "Trello Username",
```
To match the style in other preferences